recipes/jsoncpp/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.10.6" %}
+
+package:
+  name: jsoncpp
+  version: {{ version }}
+
+source:
+  fn: jsoncpp-{{ version }}.tar.gz
+  url: https://github.com/open-source-parsers/jsoncpp/archive/{{ version }}.tar.gz
+  sha256: 7c285fc40ad0c113e436a1271c4e38b5017b5c7782c306e90be9d6b2ffa90212
+
+build:
+  number: 0
+  skip: true  # [win or not py27]
```

Comment:
Tried adding `bld.bat`, but it ended up not quite working. I have left the file in case someone else would like to get it working. Have re-enabled the skip for Windows and figure this can be addressed at the feedstock.